### DISCUSSION TITLE
jpg, webp, avif in previews

### DISCUSSION THREAD

**Discussion by mantielero** [immich-app/immich#12814] [Q&A]
I see that in the configuration there is the choice between webP and jpg images. Just out of curiosity, why .avif is left out? Is it because it requires more processing power to create the previews?

**Comment by bo0tzz** [comment]
See #5117

**Reply by mantielero** [reply]
Thank you. I wasn't aware there was a feature request already.

**Comment by mertalev** [comment]
The mobile app uses Flutter and its image viewer didn't support AVIF last I checked. But I've seen someone show their AVIF originals display correctly, so maybe it does work. If not, there's also a flutter_avif package we can use.

**Reply by mantielero** [reply]
I should have checked this first. I just tried the web app and I saw it worked, so I ported all my pics and videos to .avif and av1.
